src/Arcus.Messaging.ServiceBus.Core/Extensions/MessageExtensions.cs

```diff
@@ -35,11 +35,12 @@ public static TProperty GetUserProperty<TProperty>(this Message message, string
         ///     Gets the correlation information for a given message.
         /// </summary>
         /// <param name="message">The received message.</param>
-        public static MessageCorrelationInfo GetCorrelationInfo(this Message message)
+        /// <param name="transactionIdPropertyName">Name of the property to determine the transaction id.</param>
+        public static MessageCorrelationInfo GetCorrelationInfo(this Message message, string transactionIdPropertyName = PropertyNames.TransactionId)
```

Comment:
I'm thinking, is there some place in the docs that this needs to be updated? I think there is.

Reply:
Doesn't look like it 🤔